### PR TITLE
Expressrouteur

### PR DESCRIPTION
**Pull request by Cindie1908**
Move all routes on folder routes and tests ok

### CODE REVIEW

File: back/routes/admins.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const formData = request.body;
+      connection.query('UPDATE admin SET ? WHERE id = ?', [formData, idAdmin], err => {
+      if (err) {
+        console.log(err);
+        response.status(500).send("Error editing the admin");
+      } else {
+        response.sendStatus(200);
+      }
+    });
+  });
```
Comment (rocambille) [inline back/routes/admins.js]: fix indentation

File: back/routes/places.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    if (err) {
+      console.log(err);
+      response.status(500).send("Error editing the place");
+    } else {
+      response.sendStatus(200);
+    }
+  });
+});
+
+  module.exports = router
```
Comment (rocambille) [inline back/routes/places.js:50]: fix indentation

File: back/routes/vacationers.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    if (err) {
+      console.log(err);
+      response.status(500).send("Error editing the vacationer");
+    } else {
+      response.sendStatus(200);
+    }
+  });
+});
+
+  module.exports = router
```
Comment (rocambille) [inline back/routes/vacationers.js:51]: fix indentation

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
there are conflicts to resolve in index.js

File: back/routes/admins.js (hunk 1 above)
Comment (Cindie1908) [reply back/routes/admins.js]: normalement c'est bon :)

**Pull request merged**